client/src/components/PosterStack/index.js

```diff
@@ -57,6 +57,15 @@ const PosterStack = ({
       const dirY = yDir < 0 ? -1 : 1
       const swipeThreshold = isPhoneWide ? 20 : 150
 
+      // Prevent default text-selection behavior on mobile devices
+      // while the user is swiping a card.
+      // See https://stackoverflow.com/questions/826782/how-to-disable-text-selection-highlighting
+      if (isDown) {
```

Comment:
in all honesty, I'm not sure if this is the most "react" way to do this. So take this with a grain of salt

Reply:
Actually, it looks like this is acceptable. https://stackoverflow.com/questions/47706903/add-a-class-to-the-html-body-tag-with-react

Reply:
To make this more Reacty, I could make a hook that encapsulate this behavior, but the problem is that you **cannot** call hooks inside of a callback function. So I wouldn't be able to use my custom hook inside the `bind` function